_source/_includes/blog_post.html

```diff
@@ -55,7 +55,7 @@ <h1 class="BlogPost-title p-name">
             {% assign last_updated_date = post.changelog[0] | split: '=>' | first | date: "%B %-d, %Y" %}
             <span class="vertical-spacer md-show"></span>
             <div class="BlogPost-last-updated no-excerpt">
-                Last Updated:<a href="#changelog">&nbsp;{{ last_updated_date }}</a>
+                Last Updated:&nbsp;<a href="#changelog">{{ last_updated_date }}</a>
```

Comment:
@deepu105 Is this change OK? It currently looks funny because it underlines a space when you hover over. 

<img width="715" alt="Screen Shot 2022-11-03 at 16 30 12" src="https://user-images.githubusercontent.com/17892/199846665-c086ee11-8031-4d0c-afe2-839039d9b362.png">